### ISSUE TITLE
Support for all map options

### ISSUE THREAD

**Issue by beasteers** [pauloamgomes/cockpit-cms-googlemapfield#1]
Mainly I'm interested in setting a default zoom and center, but I don't see why you couldn't just pass the field options transparently to the map constructor. There are a bunch of options that let you customize interactivity and they could be nice to have.

Time permitting, I may submit a PR.

Options API Reference: https://developers.google.com/maps/documentation/javascript/reference/map#MapOptions

Also, I'm NOT talking about having all of those options in the settings panel on the entry edit page (and having to create UI for it), but just have them globally set on the collection field type edit page (in that arbitrary json body)

**Comment by pauloamgomes** [comment]
You are right, this was implemented a long time ago and I stayed with the initial implementation. For several reasons I was not able to continue my contributions to Cockpit.. anyway, I'm happy to review a PR.

thanks

**Comment by beasteers** [comment]
no worries I figured it's been out of mind for a while haha. We'll see when I have time to get to it haha. But thanks so much for this addon!
